api/handler/delete.go

```diff
@@ -240,14 +243,6 @@ func (h *handler) DeleteMultipleObjectsHandler(w http.ResponseWriter, r *http.Re
 		}
 	}
 
-	if !requested.Quiet && len(errs) != 0 {
-		additional := []zap.Field{
-			zap.Array("objects", marshaler),
-			zap.Errors("errors", errs),
-		}
-		h.logAndSendError(w, "could not delete objects", reqInfo, nil, additional...)
-	}
-
```

Comment:
I'm not sure do we really need this codeblock because we anyway response with `DeleteObjectsResponse` struct

Reply:
It's need to log errors. Otherwise we will see errors only as response on the client side. But we can simplify condition to `if len(errs) != 0`